src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -759,4 +759,17 @@ public static String generateTimePartitionCondition(StandardTableDefinition tabl
     }
     return timePartitionCondition.toString();
   }
+
+  /**
+   * Get fully-qualified name (FQN) for a BQ table (FQN format: bigquery:{projectId}.{datasetId}.{tableId}).
+   *
+   * @param datasetProject Name of the BQ project
+   * @param datasetName    Name of the BQ dataset
+   * @param tableName      Name of the BQ table
+   * @return String fqn
+   */
+  public static String getFQN(String datasetProject, String datasetName, String tableName) {
```

Comment:
We should try to move this logic into the config class(es) if possible.

In general, broad util classes like this one are a pattern we would like to avoid. They tend to gather a bunch of unrelated code and encourage poor encapsulation.